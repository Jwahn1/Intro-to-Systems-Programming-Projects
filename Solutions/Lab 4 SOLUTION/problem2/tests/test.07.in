new 10
add hello
add world
destroy
new 5
add byebye
end
