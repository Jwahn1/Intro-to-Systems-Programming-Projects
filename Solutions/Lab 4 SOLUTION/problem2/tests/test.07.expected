(0/10) [ ]
(1/10) [ hello ]
(2/10) [ hello world ]
No list!
(0/5) [ ]
(1/5) [ byebye ]
